src/testRunners/jasmine/__tests__/JasmineFormatter-test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2016, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+'use strict';
+
+jest.autoMockOff();
+
+var jasmine = require('../../../../vendor/jasmine/jasmine-1.3.0').jasmine;
+var jasmine2Require = require('../../../../vendor/jasmine/jasmine-2.3.4.js');
+var jasmine2 = jasmine2Require.core(jasmine2Require);
+
+var JasmineFormatter = require('../JasmineFormatter');
+
+var jsdom = require('jsdom').jsdom;
+var fixture = '<html><body id="foo"></body></html>';
+
+var formatter;
+
+describe('JasmineFormatter', function() {
```

Comment:
since we are here, would you mind using arrow functions and `const` and `let` all over this file? :) I'm trying to update all the tests to newer patterns.